tests/kafkatest/tests/connect/connect_test.py

```diff
@@ -98,8 +95,6 @@ def test_file_source_and_sink(self, converter="org.apache.kafka.connect.json.Jso
         self.consumer_validator = ConsoleConsumer(self.test_context, 1, self.kafka, self.TOPIC_TEST,
                                                   consumer_timeout_ms=10000)
 
-        if self.zk:
```

Comment:
Do we need this for the cases where ZK remains? (i.e. the Kafka 2.1 case?)

or do we not test that here?